shifthelper/notifiers.py

```diff
@@ -29,8 +29,14 @@ def __init__(self,
         self.twiml = 'hangup'
 
     def notify(self, recipient, msg):
-        super().notify(recipient, msg)
-        self.not_acknowledged_calls.append((self.call, msg))
+        try:
+            super().notify(recipient, msg)
+            self.not_acknowledged_calls.append((self.call, msg))
+        except:
+            self.not_acknowledged_calls.append((None, msg))
```

Comment:
If I see that correctly we do not use the 1st tuple element at all, so this could also be:
```
self.not_acknowledged_calls.append((None, msg))
```
In both cases and further more, we refactor `not_acknowledged_calls` into `not_acknowledged_messages`